src/CampaignAdmin.php

```diff
@@ -155,6 +159,34 @@ public function readCampaigns()
         return $response;
     }
 
+    protected function getPlaceholderGroups()
+    {
+        $groups = [];
+
+        $classes = Config::inst()->get(self::class, 'placeholder_group_classes');
```

Comment:
What do you mean by this? Should we be using `ChangeSet.important_classes` instead?